src/librustc_codegen_llvm/debuginfo/metadata.rs

```diff
@@ -955,6 +957,20 @@ fn pointer_type_metadata(
     }
 }
 
+fn param_type_metadata(cx: &CodegenCx<'ll, 'tcx>, t: Ty<'tcx>) -> &'ll DIType {
+    debug!("param_type_metadata: {:?}", t);
+    let name = format!("{:?}", t);
+    return unsafe {
+        llvm::LLVMRustDIBuilderCreateBasicType(
```

Comment:
<sup>(doing this because GitHub is bad at pulling replies to old comments down to the bottom)</sup>
see https://github.com/rust-lang/rust/pull/69749#discussion_r455862518

Reply:
I've replied to the original comment.